signer/src/block_observer.rs

```diff
@@ -736,7 +741,18 @@ mod tests {
     }
 
     impl StacksInteract for TestHarness {
-        async fn get_block(&self, block_id: StacksBlockId) -> Result<NakamotoBlock, error::Error> {
+        async fn get_account(&mut self, _address: &StacksAddress) -> Result<AccountInfo, Error> {
+            todo!()
+        }
+
+        async fn submit_tx(&mut self, _tx: &StacksTransaction) -> Result<SubmitTxResponse, Error> {
+            todo!()
```

Comment:
Whenever we add TODOs like this we've been adding a comment about the ticket that is associated with this TODO. I think we should continue that practice and add such a comment here.

Reply:
Are we planning on using these functions in the block observer test harness?

Reply:
I tagged them with issue 118, which is to implement StacksInteract for the block observer

Reply:
Oh oops, you're right. I don't think we're going to implement these. No need to modify it further, we can remove the TODO and replace it with a comment about how implementing these is unnecessary later.